src/content-script.ts

```diff
@@ -1,28 +1,37 @@
+import { debug, enableDebugging } from "./debug";
+
+enableDebugging(true);
```

Comment:
I left it on purposefully; we already had it "on" to begin with since were using unqualified console.log statements. I planned on turning it off by default when there's quite a few more debug statements being generated. I am planning to be quite verbose with them, since there's a lot that can go wrong when dealing with third-party markup :)